### ISSUE TITLE
Example for runBytecodeFile not working

### ISSUE THREAD

**Issue by ednt** [OsamaAbbas/bytenode#24]
Hi,

I compiled a file test.js with `console.log('Hello World!')` inside. Nothing else.
Compilation is done by `bytenode -c test.js`

Then I use this file (testexe.js) to execute test.jsc
`const bytenode = require("bytenode");`
`bytenode.runBytecodeFile('test.jsc');`

I use
`node testexe.js`
for execution.

I get no 'Hello World!'
But if I use `bytenode test.jsc` it works.

I use node v10.13.0 in WIn10 x64

Any ideas?

**Comment by OsamaAbbas** [comment]
You have to compile using `--no-module` flag.

Your example works fine it you compile `test.js` like this:
```
$ bytenode -c test.js --no-module
````

**Comment by ednt** [comment]
Thank you for your fast response.

You are right, with this flag my simple example works.

But ...
If I use `--no-module` for my real js files, I get a 

> ReferenceError: require is not defined

 when I execute them.

To test this I extended test.js:
`const os = require('os');`
`console.log('Hello World!');`
`console.log(os.hostname());`

As soon as I have a `require` inside the error apears.

**Comment by OsamaAbbas** [comment]
OK, it is not a bug in itself, but this behavior needs to be changed so that we make your new example work, without breaking the current API.

I'll push an update to fix this issue, hopefully tomorrow or so.

Thank you.

**Comment by OsamaAbbas** [comment]
@ednt : Why don't you just use `require('./test.jsc');` in your example? Why do you use `.runBytecodeFile()` function here?

The problem is that `.runBytecodeFile()` function is used where you can't just use `require` function, as in webpack bundles or so (check #14 issue). So, if we implement `.runBytecodeFile()` function using the node require mechanism, we will break the other use case.

If you need to use `require` function inside your `.jsc` files, just compile them without `--no-module` flag, and then run them using `require('./test.jsc')`.

Please let me know what do you think.

**Comment by ednt** [comment]
Hi Osama,

sorry for the delay, but we had a public holliday and I made a 'long' weekend.

I still run into problems.
What I did:

test.js:
`const os = require('os');`
`console.log('Hello World!');`
`console.log(os.hostname());`

I compiled it with:
bytenode -c test.js --no-module

my testexe.js:
const bytenode = require("bytenode");
//bytenode.runBytecodeFile('test.jsc');
require("./test.jsc");

I run it with:
node testexe.js

I'm still not able to make it work. (And that's only a first very small test :) )
